test/integration/connect/envoy/Dockerfile-envoy-debug

```diff
@@ -0,0 +1,16 @@
+ARG ENVOY_VERSION
+
+FROM docker.mirror.hashicorp.services/envoyproxy/envoy-debug:v${ENVOY_VERSION}
+
+RUN apt-get update -y
+RUN apt-get install -y --no-install-recommends python3 binutils
```

Comment:
- `python3` : the wrapper script is a python script
- `binutils` : gives us `addr2line` which is needed by the script